packages/pcsx-rearmed/configure.patch

```diff
@@ -0,0 +1,58 @@
+--- a/configure	2022-03-14 19:04:27.000000000 -0300
++++ b/configure	2023-01-13 15:18:19.498029946 -0300
+@@ -2,12 +2,14 @@
+ # some elements originated from qemu configure
+ set -e
+ 
+-TMPC="/tmp/pcsx-conf-${RANDOM}-$$-${RANDOM}.c"
+-TMPO="/tmp/pcsx-conf-${RANDOM}-$$-${RANDOM}.o"
+-TMPB="/tmp/pcsx-conf-${RANDOM}-$$-${RANDOM}"
++TMPC="${PREFIX}/tmp/pcsx-conf-${RANDOM}-$$-${RANDOM}.c"
++TMPO="${PREFIX}/tmp/pcsx-conf-${RANDOM}-$$-${RANDOM}.o"
++TMPB="${PREFIX}/tmp/pcsx-conf-${RANDOM}-$$-${RANDOM}"
+ trap "rm -f $TMPC $TMPO $TMPB" EXIT INT QUIT TERM
+ rm -f config.log
+ 
++CFLAGS="${CFLAGS} -I${PREFIX}/include"
++
+ compile_object()
+ {
+   c="$CC $CFLAGS -c $TMPC -o $TMPO $@"
+@@ -69,13 +71,13 @@
+ CXX="${CXX-${CROSS_COMPILE}g++}"
+ AS="${AS-${CROSS_COMPILE}as}"
+ AR="${AS-${CROSS_COMPILE}ar}"
+-MAIN_LDLIBS="$LDLIBS -ldl -lm -lpthread"
++MAIN_LDLIBS="$LDLIBS -ldl -lm -lc"
+ config_mak="config.mak"
+ 
+ fail()
+ {
+   echo "$@"
+-  if test -n "$DUMP_CONFIG_LOG"; then cat config.log; fi
++  cat config.log
+   exit 1
+ }
+ 
+@@ -398,7 +400,7 @@
+     sound_drivers="$sound_drivers pulseaudio"
+     MAIN_LDLIBS="-lpulse $MAIN_LDLIBS"
+   fi
+-  if [ "$need_sdl" = "yes" ] || check_sdl `sdl-config --cflags --libs`; then
++  if [ "$need_sdl" = "yes" ] || check_sdl -I/data/data/com.termux/files/usr/include/SDL -lSDL; then
```

Comment:
Do not hardcode the value of `PREFIX` in patches. Use `@TERMUX_PREFIX@` instead.

Alternatively it is suggested to use `@TERMUX_PREFIX@/bin/sdl-config` in place of `sdl-config`. (FYI, `*-config` scripts are properly handled when using Autotools but otherwise not.)